### ISSUE TITLE
Not Bug - Please Remove Personal Content From Archive

### ISSUE THREAD

**Issue by myhero34** [eksopl/fuuka#86]
Hello,

I have tried emailing every contact I could find on the warosu.org website, including admin@warosu.org, but could not get my content deleted.

A few years ago my friend thought it would be funny to impersonate me on several boards acting like a novice wanting to get involved in whatever board he thought would look bad (he posted on /hm/, /jp/, you get the idea). 

Little did he know, most of these boards archive their posts and because of that are some of the first results when I google my name. This is sort of a disastrous situation because of how unique my name is and because of how important a good online reputation is in finding a job.

Basically, all I need is for one of his posts to be taken down from your archive. No one is viewing the thread or finding it interesting, and it contains some of my personal information. I have asked other archives to do the same and they have obliged. I have reported the post a few times before and tried using the website's delete function but I suppose it hasn't been deleted because it doesn't break any laws.

Here is the post: https://warosu.org/jp/thread/10422937

Thanks and have a good day!

**Comment by GXTX** [comment]
The email is haha@warosu.org, you would have figured that out if you just looked at the top of their website.

**Comment by myhero34** [comment]
Hi, 

Thanks for the reply but that was the first email I tried.
I emailed haha on the 13th. I suppose my email could have counted as spam, but I don't really know a way to circumvent that.
The post is still the fourth result that comes up when you google my full name, which is unfortunately what people are going to google after receiving my resume.

Any help would be appreciated and I apologize for using github incorrectly.
